frontend/src/metabase/querying/utils/drills/combine-columns-drill/components/ColumnPicker/ColumnPicker.tsx

```diff
@@ -0,0 +1,153 @@
+import classNames from "classnames";
+import type { FocusEvent, MouseEvent, KeyboardEvent, ReactNode } from "react";
+import { useRef, useState, useMemo, useEffect } from "react";
+import { t } from "ttag";
+
+import { QueryColumnPicker } from "metabase/common/components/QueryColumnPicker";
+import useSequencedContentCloseHandler from "metabase/hooks/use-sequenced-content-close-handler";
+import { color } from "metabase/lib/colors";
+import { Button, Icon, Input, Popover, FocusTrap } from "metabase/ui";
+import { getThemeOverrides } from "metabase/ui/theme";
+import * as Lib from "metabase-lib";
+
+import styles from "./ColumnPicker.module.css";
+
+type ColumnInputProps = {
+  query: Lib.Query;
+  stageIndex: number;
+  columns: Lib.ColumnMetadata[];
+  label?: string;
+  value: Lib.ColumnMetadata;
+  onChange: (column: Lib.ColumnMetadata) => void;
+};
+
+const theme = getThemeOverrides();
```

Comment:
Strong no. This should never be used directly.

Reply:
I followed this pattern because it is already in use in the codebase.

I've change the code to use `useMantineTheme` but I had to export it from `metabase/ui` because it wasn't yet.

Or do you have another preferred way of reading theme properties?

Reply:
`useMantineTheme` LGTM